tests/test_basic.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os
+
+
+def test_oda_api_code(dispatcher_live_fixture):
+    from oda_api.api import DispatcherAPI
+    disp = DispatcherAPI(
+        url=dispatcher_live_fixture
+    )
+
+    disp.get_product(
+        product_type="Dummy",
+        instrument="empty",
+        product="dummy"
+    )
+
+    job_id = disp.job_id
+    session_id = disp.session_id
+    scratch_path = f'scratch_sid_{session_id}_jid_{job_id}'
+    scratch_path_aliased = f'scratch_sid_{session_id}_jid_{job_id}_aliased'
+    assert os.path.exists(scratch_path) or os.path.exists(scratch_path_aliased)
+
+    if os.path.exists(scratch_path):
+        f_query_output = open(scratch_path + '/query_output.json')
+        f_params = open(scratch_path + '/analysis_parameters.json')
+    else:
+        f_query_output = open(scratch_path_aliased + '/query_output.json')
+        f_params = open(scratch_path_aliased + '/analysis_parameters.json')
+
+    jdata_output = json.load(f_query_output)
+
+    assert 'prod_dictionary' in jdata_output
+    assert 'api_code' in jdata_output['prod_dictionary']
+
+    output_api_code = jdata_output['prod_dictionary']['api_code']
+
+    param_dic = json.load(f_params)
```

Comment:
this till line 52 all a copy of the oda_api code, right? Can be removed and expectation spelled out, since you know what you expect.

Reply:
Ok, next commit makes the whole expected api output code explicit and not inferred/calculated. And it's lot less code :) Thanks for the suggestion